.github/workflows/publish-to-pypi.yml

```diff
@@ -1,9 +1,6 @@
 name: Publish to PyPi
 
-on:
-  pull_request:
-    branches:
-    - master
+on: push
```

Comment:
Maybe we can have this be also a workflow dispatch so that we can run this whenever we want

Reply:
Agreed